src/main/java/org/opentripplanner/netex/loader/parser/ResourceFrameParser.java

```diff
@@ -67,4 +65,16 @@ private void parseOrganization(Organisation_VersionStructure element) {
             warnOnMissingMapping(LOG, element);
         }
     }
+
+    private void parseBrandings(TypesOfValueInFrame_RelStructure element) {
+        if (element == null || element.getValueSetOrTypeOfValue() == null) {
+            return;
+        }
+
+        for (JAXBElement<?> e : element.getValueSetOrTypeOfValue()) {
+             if (e.getValue() instanceof Branding) {
+                brandings.add((Branding) e.getValue());
+             }
+        }
+    }
```

Comment:
1. This method should probably have another name, it parses many diffrent types, not just branding. 
2. The `if` in the for loop should probably have an `else` with `warnOnMissingMapping(LOG, e);`

Reply:
Changed name and added the missing else statement with warning.